services/storage/tests/unit/modules/celery/test_celery.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import time
+from collections.abc import Callable
+from random import randint
+
+import pytest
+from celery import Celery, Task
+from celery.contrib.abortable import AbortableTask
+from simcore_service_storage.main import CeleryTaskQueueClient
+from simcore_service_storage.modules.celery.example_tasks import sync_archive
```

Comment:
if you are just using this one for testing. just move it here. I will add the one I need based on this example